tests/kernel/bigquery_tests.py

```diff
@@ -46,6 +46,35 @@ def noop_decorator(func):
 
 class TestCases(unittest.TestCase):
 
+  def test_table_schema(self):
+    import jsonschema
+    schemas = [
+      {
+        'schema': [
+          {'name': 'col1', 'type': 'int64', 'mode': 'NULLABLE', 'description': 'description1'},
+          {'name': 'col1', 'type': 'STRING', 'mode': 'required', 'description': 'description1'}
+        ]
+      },
+      {
+        'schema': [
+          {'name': 'col1', 'type': 'record', 'mode': 'repeated', 'description': 'description1',
+           'fields': [
+             {'name': 'field1', 'type': 'int64'},
+             {'name': 'field2', 'type': 'int64'},
+             {'name': 'field3', 'type': 'record',
+              'fields': [
+                {'name': 'nestedField1', 'type': 'STRING'},
+                {'name': 'nestedField2', 'type': 'STRING'}
+              ]}
+           ]}
+        ]
+      }
+    ]
+
+    for s in schemas:
+      record = google.datalab.utils.commands.parse_config(json.dumps(s), {})
+      jsonschema.validate(record, google.datalab.bigquery.commands._bigquery.table_schema_schema)
```

Comment:
This makes sure some good scenarios are valid. Schemas with nested fields were broken until #430, so this is a regression test.